src/EVault/shared/Base.sol

```diff
@@ -82,7 +82,7 @@ abstract contract Base is EVCClient, Cache {
         // For this reason, the snapshot is disabled if both caps are disabled.
         if (
             !vaultCache.snapshotInitialized
-                && (vaultCache.supplyCap < type(uint256).max || vaultCache.borrowCap < type(uint256).max)
+                && !(vaultCache.supplyCap == type(uint256).max && vaultCache.borrowCap == type(uint256).max)
```

Comment:
I think this change is redundant with the commit in #175 ?

Reply:
fixed, commit removed